phoenix-core/src/it/java/org/apache/phoenix/end2end/DerivedTableIT.java

```diff
@@ -313,21 +290,25 @@ public void testDerivedTableWithGroupBy() throws Exception {
             query = "SELECT DISTINCT COLLECTDISTINCT(t.b) FROM (SELECT b_string b, a_string a FROM aTable GROUP BY a_string, b_string) AS t GROUP BY t.a";
```

Comment:
@maryannxue - do you think the underlying order returned is indeterminate? For our tests, we have phoenix.query.force.rowkeyorder set to true, so underlying scans will return in row key order. Just want to make sure @chiastic-security fix isn't masking a bug.